plugins/vpp/ifplugin/descriptor/interface.go

```diff
@@ -366,6 +369,9 @@ func (d *InterfaceDescriptor) Validate(key string, intf *interfaces.Interface) e
 			return kvs.NewInvalidValueError(ErrSubInterfaceWithoutParent, "link.sub.parent_name")
 		}
 	case interfaces.Interface_DPDK:
+		if _, ok := d.ethernetIfs[intf.Name]; !ok {
+			return kvs.NewInvalidValueError(ErrDPDKInterfaceMissing, "link.sub.parent_name")
```

Comment:
after merge I noticed that the name of the parameter is wrong: please change "link.sub.parent_name" to "name"

Reply:
here: https://github.com/ligato/vpp-agent/commit/4c9b099928726feea6d32b12420e84daa7a84f97